packages/edit-site/src/components/sidebar-navigation-screen-pattern/use-pattern-details.js

```diff
@@ -82,5 +89,40 @@ export default function usePatternDetails( postType, postId ) {
 		/>
 	) : null;
 
-	return { title, description, footer };
+	const details = [];
+
+	if ( postType === 'wp_block' ) {
+		details.push( {
+			label: __( 'Syncing' ),
+			value:
+				record.meta?.sync_status === 'unsynced'
+					? __( 'Not synced' )
+					: __( 'Fully synced' ),
+		} );
+	}
+
+	const content = (
+		<>
+			{ !! details.length && (
+				<SidebarNavigationScreenDetailsPanel
+					spacing={ 5 }
+					title={ __( 'Details' ) }
+				>
+					{ details.map( ( { label, value } ) => (
+						<SidebarNavigationScreenDetailsPanelRow key={ label }>
+							<SidebarNavigationScreenDetailsPanelLabel>
+								{ label }
+							</SidebarNavigationScreenDetailsPanelLabel>
+							<SidebarNavigationScreenDetailsPanelValue>
+								{ value }
+							</SidebarNavigationScreenDetailsPanelValue>
+						</SidebarNavigationScreenDetailsPanelRow>
+					) ) }
+				</SidebarNavigationScreenDetailsPanel>
+			) }
+			{ useNavigationMenuContent( postType, postId ) }
```

Comment:
Should this convert to a component? I just came across this pattern and feel odd 😅